SimGeneral/CaloAnalysis/plugins/CaloTruthAccumulator.cc

```diff
@@ -223,6 +229,7 @@ CaloTruthAccumulator::CaloTruthAccumulator(const edm::ParameterSet &config,
     : messageCategory_("CaloTruthAccumulator"),
       maximumPreviousBunchCrossing_(config.getParameter<unsigned int>("maximumPreviousBunchCrossing")),
       maximumSubsequentBunchCrossing_(config.getParameter<unsigned int>("maximumSubsequentBunchCrossing")),
+      bunchSpacing_(config.getParameter<unsigned int>("bunchspace")),
```

Comment:
Hi, just a small comment—I don't see the usage of bunchSpacing. Maybe I'm missing something? Thanks

Reply:
Yes sorry, it was a leftover from previous changes. I've removed it.